llmfoundry/data/packing.py

```diff
@@ -367,6 +379,8 @@ def auto_packing_ratio(
         num_packing_ratios=num_packing_ratios,
         device_batch_size=device_batch_size,
     )
+    dist.barrier()
```

Comment:
are you planning to keep this dist barrier?

Reply:
Removed